libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -184,9 +188,25 @@ const AP_Param::GroupInfo NavEKF3::var_info[] = {
     // @Units: m
     AP_GROUPINFO("GLITCH_RAD", 7, NavEKF3, _gpsGlitchRadiusMax, GLITCH_RADIUS_DEFAULT),
 
-    // 8 previously used for EKF3_GPS_DELAY parameter that has been deprecated.
-    // The EKF now takes its GPS delay form the GPS library with the default delays
-    // specified by the GPS_DELAY and GPS_DELAY2 parameters.
+    // bluff body drag fusion
+
+    // @Param: BCOEF_X
+    // @DisplayName: Ballistic coefficient measured in X direction
+    // @Description: Ratio of mass to drag coefficient measured along the X body axis. This parameter enables estimation of wind drift for vehicles with bluff bodies and without propulsion forces in the X and Y direction (eg multicopters). The drag produced by this effect scales with speed squared.  Set to a postive value > 1.0 to enable. A starting value is the mass in Kg divided by the frontal area. The predicted drag from the rotors is specified separately by the EK3_MCOEF parameter.
+    // @Range: 0.0 1000.0
+    // @User: Advanced
+
+    // @Param: BCOEF_Y
+    // @DisplayName: Ballistic coefficient measured in Y direction
+    // @Description: Ratio of mass to drag coefficient measured along the Y body axis. This parameter enables estimation of wind drift for vehicles with bluff bodies and without propulsion forces in the X and Y direction (eg multicopters). The drag produced by this effect scales with speed squared.  Set to a postive value > 1.0 to enable. A starting value is the mass in Kg divided by the side area. The predicted drag from the rotors is specified separately by the EK3_MCOEF parameter.
+    // @Range: 50.0 1000.0
+    // @User: Advanced
+
+    // @Param: BCOEF_Z
+    // @DisplayName: unused
```

Comment:
this leaves unused params. We could add AP_Vector2f or just do separate X and Y params